interop/test-runner/main.go

```diff
@@ -530,22 +546,122 @@ func (config *ScriptActorConfig) ChangeCipherSuite() error {
 func (config *ScriptActorConfig) RunStep(index int, step ScriptStep) error {
 	switch step.Action {
 	case ActionCreateGroup:
-		client := config.ActorClients[step.Actor]
-		groupID := []byte(uuid.New().String())
-		req := &pb.CreateGroupRequest{
-			GroupId:          groupID,
-			CipherSuite:      config.CipherSuite,
-			EncryptHandshake: config.EncryptHandshake,
-			Identity:         []byte(step.Actor),
-		}
-		resp, err := client.rpc.CreateGroup(ctx(), req)
+		var params CreateGroupStepParams
+		err := json.Unmarshal(step.Raw, &params)
 		if err != nil {
 			return err
 		}
-		config.Log(step.Actor, "CreateGroup", req, resp)
 
-		config.stateID[step.Actor] = resp.StateId
-		config.StoreMessage(index, "group_id", groupID)
+		// Create the group
+		groupID := []byte(uuid.New().String())
+		{
+			client := config.ActorClients[step.Actor]
+			req := &pb.CreateGroupRequest{
+				GroupId:          groupID,
+				CipherSuite:      config.CipherSuite,
+				EncryptHandshake: config.EncryptHandshake,
```

Comment:
Somewhat orthogonal, but why is `EncryptHandshake` a property of the config, while e.g. `ForcePath` is a property of a step parameter? Should `EncryptHandshake` also be specified per step?

Reply:
I think the main idea here is that having `EncryptHandshake` as a `config` parameter makes it easy to automatically test with and without it across all messages.

Reply:
OK, I see how it's somewhat different.